src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -87,13 +88,50 @@ const OwnerPanel = (props: Props) => {
           </button>
         )}
       </div>
+      {Object.keys(props.extraInfo).length !== 0 ?
```

Comment:
I think if/else would be easier to read here

Reply:
sorry, can you clarify? i thought jsx doesn't like if/else?